BlueprintUI/Sources/BlueprintView/BlueprintView.swift

```diff
@@ -525,6 +527,13 @@ public final class BlueprintView: UIView {
 
         var environment = inherited.merged(prioritizing: environment)
 
+        /// We're a root `BlueprintView`, so pass in the view measurer for our downstream
+        /// dependencies if we didn't inherit a measurement cache.
+
+        if environment.inheritedViewCache == nil {
+            environment.viewCache = viewCache
+        }
```

Comment:
I'm still a little nervous about cascading view caches through the environment. Won't it still leave open the possibility that nested blueprint view elements leak stuff through long(er)-lived caches?

Reply:
It's definitely possible, but largely the root blueprint view lives on the screen level.